client/header/activity-panel/panels/inbox/index.js

```diff
@@ -175,6 +175,7 @@ export default compose(
 		const inboxQuery = {
 			page: 1,
 			per_page: QUERY_DEFAULTS.pageSize,
+			status: 'unactioned',
```

Comment:
What do you think about extracting these query params to a common function (in a followup PR) that both the inbox and the unread indicator can share? It seems like they always should be in-sync.

Reply:
Sure